mlflow/Dockerfile

```diff
@@ -3,8 +3,10 @@ FROM python:3.7.2-slim
 RUN pip install google-cloud-storage
 RUN pip install psycopg2-binary
 
+ARG BOTO3_VERSION=1.7.12
```

Comment:
Why is it needed?

Reply:
So MLflow can use minio as artifact storage